public/locales/ru/translations.json

```diff
@@ -2,52 +2,52 @@
   "language name": "Русский",
 
   "home": "Главная",
-  "recent": "Недавняя активность",
-  "no followers": "Нет подписчиков",
-  "latest followers": "Последние подписчики",
+  "recent": "Недавняя Активность",
+  "no followers": "Ни одного подписчика",
+  "latest followers": "Последние Подписавшиеся",
   "connect wallet to view": "Подключите кошелек для просмотра",
   "phrase": "Все дело в том, кого ты знаешь. 😜",
   "social graph first": "Социальный граф",
-  "social graph second": "для Ethereum.",
+  "social graph second": "для сети Ethereum.",
   "follow ghost first": "Следите за друзьями.",
   "follow ghost second": "Избегайте врагов.",
   "follower list first": "Список подписок",
   "follower list second": "которым вы владеете.",
   "integrations": "Интеграции",
   "partners": "Партнеры запуска",
   "sponsors": "Поддерживается грантами от:",
-  "support": "Поддержка",
+  "support": "Служба Поддержки",
 
   "profile": "Мой Профиль",
   "profile error": "Пользователя не существует",
   "settings": "Настройки Списка",
   "block-mute": "Списки Блок./Игнор.",
   "edit settings": "Изменить Настройки",
-  "edit profile": "Изменить профиль",
-  "top eight title": "🔥 Мой Топ 8 🔥",
-  "no top eight": "Здесь ничего нет... пока еще 😳",
-  "top eight limit": "Ваш Топ 8 заполнен! Чтобы добавить нового человека, сперва удалите кого-то. 😈",
-  "top eight description": "Добавляйте друзей в свой Топ 8! Вы всегда сможете добавить или удалить людей позже.",
+  "edit profile": "Редактировать",
```

Comment:
Единственный случай, когда необходимо не уточнение, а упрощение. Почти все, кто в русскоязычном пространстве сейчас пользуется интернетом на продвинутом уровне и является целевой аудиторией EFP, пользовались VK до глобального изменения UI. Да и после этого периода, все платформы где ты имеешь редактируемый профиль, используют слово «Редактировать» вместо «Изменить Профиль». «Редактировать/Править» – для личной информации, которую видят пользователи, «Изменить/Настроить» – для каких-либо настроек, например конфиденциальности. 


The only case when it is necessary not to clarify, but to simplify. Almost everyone who in the Russian-speaking space now uses the Internet at an advanced level and is the target audience of EFP, used VK before the global UI change. And even after this period, all platforms where you have an editable profile use the word «Редактировать» instead of «Изменить профиль». «Редактировать/Править» – for personal information that users see, «Изменить/Настроить» – for any settings, such as privacy.

Reply:
"Редактировать" подразумевает, что на этом сайте откроется редактор. В нашей ситуации это ссылка на другой сайт.

Reply:
Да, ты прав. Не учел этот момент. Давай оставим "Изменить профиль".